compiler/generator/rust/rust_instructions.hh

```diff
@@ -287,7 +287,7 @@ class RustInstVisitor : public TextInstVisitor {
             // impls, we need a mechanism to forward the information whether to use "pub"
             // or not. In the worst case, we have to prefix the name string like "pub fname",
             // and handle the prefix here.
-            *fOut << "fn " << inst->fName;
+            *fOut << "pub fn " << inst->fName;
```

Comment:
What is this change about? (I couldn't really figure out what it affects during the code gen.)

Is the comment actually still valid after this change, because now function are no longer only attached to a trait as the first sentence says?

Reply:
i think it influenced the generation of get_num_inputs() before i changed it in the other pr.
i would remove the pub again.
Since functions are not attached it wouldn't matter if they are prefixed with "pub".
But because this is for internal functions we don't need pub.
I don't think the comment makes sense so I would remove it completely.